### DISCUSSION TITLE
DigitalOcean .. One Click vs App platform

### DISCUSSION THREAD

**Discussion by eliteproxy7** [directus/directus#5771] [Archive]
the one click install is fast and simple but i would like to use the DigitalOcean App platform. When I try to do this i get this error
![image](https://user-images.githubusercontent.com/5486268/118889785-9d306b00-b8cb-11eb-84bd-6ea5f0fad2b2.png)

I am almost certain I may be doing something wrong with the env variables?

**Comment by rijkvanzanten** [comment]
> I am almost certain I may be doing something wrong with the env variables?

Plausible! Did you follow https://docs.directus.io/guides/installation/digitalocean-app-platform/? The last part is the crucial step, as it needs the CA CERT in order to connect properly from App Platform.
